### DISCUSSION TITLE
Welcome to CryptoTracker Discussions!

### DISCUSSION THREAD

**Discussion by ismaelestalayo** [ismaelestalayo/CryptoTracker#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sopwith** [comment]
Feature request: add the ability to set alerts for when a currency reaches a particular price, or exceeds / drops below it. :)

**Reply by ismaelestalayo** [reply]
hey @Sopwith thanks for your suggestion.
While I havent begin with it, Price alerts are indeed in my to-do list for the next 6.0 update that I'm working on.

If you want to see what I've done so far and my current plans, you can check out the [v6.0 roadmap](https://github.com/ismaelestalayo/CryptoTracker/projects/1)

**Comment by DandyHry** [comment]
Hi, its possible to set alerts worker for a shorter period than 15 minutes? Ideal to 1 minute, or make a option in settings window. In crypto is 15 minutes to long time :)
